ports/libsimpleservo/api/src/lib.rs

```diff
@@ -164,6 +169,7 @@ pub struct ServoGlue {
     browsers: Vec<BrowserId>,
     events: Vec<WindowEvent>,
     current_url: Option<ServoUrl>,
+    context_menu_sender: Option<IpcSender<ContextMenuResult>>,
```

Comment:
I *think* this could be a threaded channel. The `EmbedderProxy` itself also uses a threaded channel internally.